rasa/nlu/classifiers/keyword_intent_classifier.py

```diff
@@ -86,13 +88,14 @@ def _validate_keyword_map(self) -> None:
                     and intent1 != intent2
                 ):
                     ambiguous_mappings.append((intent1, keyword1))
-                    warnings.warn(
+                    raise_warning(
                         f"Keyword '{keyword1}' is a keyword of intent '{intent1}', "
                         f"but also a substring of '{keyword2}', which is a "
                         f"keyword of intent '{intent2}."
                         f" '{keyword1}' will be removed from the list of keywords.\n"
                         "Remove (one of) the conflicting keywords from the"
```

Comment:
```suggestion
                        f"Remove (one of) the conflicting keywords from the"
```

Reply:
looks like we're adding "f" for all sub-lines of multi strings, so we need it here as well.